tests/TestCase/Cache/Engine/ApcEngineTest.php

```diff
@@ -286,4 +286,24 @@ public function testGroupClear()
         $this->assertTrue(Cache::clearGroup('group_b', 'apc_groups'));
         $this->assertFalse(Cache::read('test_groups', 'apc_groups'));
     }
+
+    /**
+     * Test that failed add write return false.
```

Comment:
The title is misleading. Not ony the false case is tested, but also the true case.

Titles of the test methods seem to be inconsistent, too:
https://github.com/cakephp/cakephp/pull/7206/files#diff-584b46532c066ec504d65b7e6f624f08R564